### PR TITLE
Just a couple of optimizations and style changes to PR #25.

### PR DESCRIPTION
**Pull request by cvallance**
An optimisation and a few style changes to fit with the the project.

### CODE REVIEW

**Comment by cvallance** [comment]
CC @v-antech & @jwaldrip

File: src/lib/config.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+var dns = require('dns');
```
Comment (cvallance) [inline src/lib/config.js]: Prefer to put all requires at the top

File: src/lib/config.js [hunk 2]
```diff
@@ -31,25 +33,36 @@ var getK8sClusterDomain = function() {
...
-        }
-      });
-    }
+/**
+ * Calls a reverse DNS lookup to ensure that the given custom domain name matches the actual one.
+ * Raises a console warning if that is not the case.
+ * @param clusterDomain the domain to verify.
+ */
+var verifyCorrectnessOfDomain = function(clusterDomain) {
+  if (!clusterDomain) {
```
Comment (cvallance) [inline src/lib/config.js]: Prefer return early on negative if clauses vs nesting positive ones

Note: there's a couple of these that I've changed

File: src/lib/config.js [hunk 3]
```diff
@@ -31,25 +33,36 @@ var getK8sClusterDomain = function() {
...
+    return;
   }
+
+  var servers = dns.getServers();
```
Comment (cvallance) [inline src/lib/config.js]: Only calling `dns.getServers()` once

**Comment by v-antech** [comment]
First of all Thanks! :)
The code changes look fine, still I deployed it on our test cluster and did some testing - deleting pods, adding pods, etc. Seemed to work.

**Comment by admin-labs** [comment]
Hello Charles - hope all is well.

I'm wondering why I see email communications from you to the
admin-labs@leportschools.com account? Are you still subscribed to our labs
github account?

Please let us know.


Thanks.

**Pull request merged**